### PR TITLE
Adjust latency limits for large clusters for listing pods

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

File: test/e2e/metrics_util.go [hunk 1]
```diff
@@ -40,7 +40,9 @@ import (
 
 const (
 	podStartupThreshold           time.Duration = 5 * time.Second
-	listPodLatencyThreshold       time.Duration = 2 * time.Second
+	listPodLatencySmallThreshold  time.Duration = 2 * time.Second
+	listPodLatencyMediumThreshold time.Duration = 4 * time.Second
+	listPodLatencyLargeThreshold  time.Duration = 15 * time.Second
```
Comment (wojtek-t) [inline test/e2e/metrics_util.go]: It should scale linearly with number of nodes - so I would say it should be 8.

Comment (gmarek) [inline test/e2e/metrics_util.go]: We should, but currently we're seeing low teens there. I'd say let's start with 15 and work our way down to 8.

Comment (wojtek-t) [inline test/e2e/metrics_util.go]: We should down all of them to 1s in the fullness of time (not to 8) :P 

Can you please add a TODO to decrease those limits?

Comment (gmarek) [inline test/e2e/metrics_util.go]: Done.

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**